ignite/pkg/protoanalysis/protoutil/creator_test.go

```diff
@@ -0,0 +1,577 @@
+package protoutil
+
```

Comment:
Can you define the test cases and structs inside the test functions like [here](https://github.com/ignite/cli/blob/d23b5e7847e2bcd3141aef10b9991629a661cc2e/ignite/pkg/cosmosgen/cosmosgen_test.go#L15)?  This is the convention we use for table-driven tests in the codebase.

https://dave.cheney.net/2019/05/07/prefer-table-driven-tests for some more examples.